### PR TITLE
Hw

### PR DESCRIPTION
**Pull request by mrmax24**

### CODE REVIEW

**Review by MentorMA — APPROVED** [review]

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    @Column(nullable = false)
+    private BigDecimal price;
+
+    private String description;
+
+    private String coverImage;
+
+    @Column(nullable = false)
+    private boolean isDeleted = false;
+}
```
Comment (MentorMA) [inline src/main/java/mate/academy/bookstore/model/Book.java]: Keep one style between fields

Comment (MentorMA) [reply src/main/java/mate/academy/bookstore/model/Book.java]: image

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/bookstore/config/MapperConfig.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
+
+)
+
```
Comment (okuzan) [inline src/main/java/mate/academy/bookstore/config/MapperConfig.java:13]:
```suggestion
)
```

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
+@Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```
Comment (okuzan) [inline src/main/java/mate/academy/bookstore/model/Book.java:17]:
```suggestion
@SQLDelete(sql = "UPDATE books SET is_deleted = TRUE WHERE id = ?")
@Where(clause = "is_deleted = FALSE")
```

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+@Table(name = "books")
+public class Book {
+
```
Comment (okuzan) [inline src/main/java/mate/academy/bookstore/model/Book.java:20]:
```suggestion
```
no empty lines after method/class definition are required

**Review by okuzan — CHANGES_REQUESTED** [review]

**Pull request merged**